### ISSUE TITLE
There is a branch named 2.1.0 and a tag named 2.1.0 and they point at different commits

### ISSUE THREAD

**Issue by ruuda** [near/nearcore#12016]
In this repository, there is currently a branch named `2.1.0` and it points at 88075cec93923701af9fc43760aa41f3fefffa3f, and there is a tag `2.1.0` and it points at 24dbd5f8c2bf6071efd4be00fc0c15af5449b6b3. To confirm this for yourself:

```
$ git clone https://github.com/near/nearcore.git
$ cd nearcore
$ git rev-parse refs/remotes/origin/2.1.0
88075cec93923701af9fc43760aa41f3fefffa3f

$ git rev-parse refs/tags/2.1.0
24dbd5f8c2bf6071efd4be00fc0c15af5449b6b3
```

This creates confusion about what 2.1.0 means. Probably the tag is intended to be the source of truth, but without the qualified name, it’s not clear which one Git uses.

The best way to fix this is probably to delete the 2.1.0 branch; the tag is already there.

**Comment by jancionear** [comment]
You're right, it looks like the commits meant for `2.1.1` made it onto the `2.1.0` branch by accident.
I think we should remove the last three commits from `2.1.0`, and force push a branch that ends at https://github.com/near/nearcore/commit/24dbd5f8c2bf6071efd4be00fc0c15af5449b6b3
/cc @VanBarbascu

**Comment by qezz** [comment]
So when one does `git checkout 2.1.0`, will it eventually end up on the branch or on the tag?

**Comment by jancionear** [comment]
> So when one does `git checkout 2.1.0`, will it eventually end up on the branch or on the tag?

Both, the tag and the branch will point to the same commit

**Comment by qezz** [comment]
> Both, the tag and the branch should point to the same commit

Well, it should, but Git doesn't enforce it. So people may end up thinking that they are building from the same commit, but in reality it's not the case.

**Comment by telezhnaya** [comment]
Thanks for finding this! Fixed, now they both point to 24dbd5f8c2bf6071efd4be00fc0c15af5449b6b3
